keylime/cmd_exec.py

```diff
@@ -1,31 +1,42 @@
 import os
 import subprocess
 import time
+from typing import Any, Dict, List, Optional, Sequence, Tuple, Union, cast
 
 EXIT_SUCESS = 0
 
+EnvType = Dict[str, str]
 
-def _execute(cmd, env=None, **kwargs):
+
+def _execute(cmd: Sequence[str], env: Optional[EnvType] = None, **kwargs) -> Tuple[bytes, bytes, int]:
     with subprocess.Popen(cmd, env=env, stdout=subprocess.PIPE, stderr=subprocess.PIPE, **kwargs) as proc:
         out, err = proc.communicate()
-        return out, err, proc.returncode
+        # All callers assume to receive a list of bytes back
+        return cast(bytes, out), cast(bytes, err), proc.returncode
```

Comment:
This only holds when text mode is never set to True. We should add a comment indicating that.

Reply:
Added that text mode is not being used to the comment now.